pkg/codegen/testing/test/testdata/inline-invokes-pp/go/inline-invokes.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		invokeLookupVpc, err := ec2.LookupVpc(ctx, &ec2.LookupVpcArgs{
+			Default: pulumi.BoolRef(true),
+		}, nil)
+		if err != nil {
+			return err
+		}
+		_, err = ec2.NewSecurityGroup(ctx, "webSecurityGroup", &ec2.SecurityGroupArgs{
+			VpcId: invokeLookupVpc.Id,
```

Comment:
Something is off here where the expression `invokeLookupVpc.Id` should actually be `pulumi.String(invokeLookupVpc.Id)` not entirely sure why the conversion isn't applied after the extracting the invoke 🤔 this is why go is skipping compilation at the moment

Reply:
That sounds a lot like the error for https://github.com/pulumi/pulumi/issues/14339